API/Repositories/Implementations/PlayerRepository.cs

```diff
@@ -163,13 +163,13 @@ private IQueryable<Player> Search(string username)
             return _context.Players.Where(p =>
                 p.Username != null
                 && (
-                    p.Username.Equals(username, StringComparison.CurrentCultureIgnoreCase)
-                    || p.Username.Equals(username, StringComparison.CurrentCultureIgnoreCase)
+                    EF.Functions.Like(p.Username, username)
+                    || EF.Functions.Like(p.Username, username)
```

Comment:
[See here ](https://stackoverflow.com/a/45709434), I think you need a wildcard (`%`) directly after the search input, and you probably have to do .ToLower(), not sure.

If you find this is not the case in local testing, lmk.

(This applies everywhere `EF.Functions.Like` is used)

Reply:
Technically case sensitivity on lookups on a column should be handled by collations at the database level, we can use .ToLower() but it could have performance impacts later on from what I understand.